warehouse/query-core/src/test/java/datawave/query/index/lookup/RangeStreamTest.java

```diff
@@ -1579,7 +1578,7 @@ public void testIntersection_ofDayRangesAndShardRange() throws Exception {
         
         config.setQueryFieldsDatatypes(dataTypes);
         config.setIndexedFields(dataTypes);
-        config.setShardsPerDayThreshold(2);
+        config.setShardsPerDayThreshold(0); // set to zero to roll up to day ranges
```

Comment:
How is changing this from 2 to 0 actually testing anything new?

Reply:
Now that we actually do more seeking, we need to lower the shards per day threshold to hit the rollup.